tests/integration/test_vulnerability_detector/test_windows/test_cpe_indexing.py

```diff
@@ -157,7 +157,7 @@ def mock_system(request):
     control_service('stop', daemon='wazuh-db')
 
     # Clean tables
-    vd.clean_vd_tables(agent='000')
+    vd.clean_vd_tables(agent=mock_agent)
 
     control_service('start', daemon='wazuh-db')
 
```

Comment:
Hardcoded timeouts in log monitor calls are not allowed. It should be good to declare a global variable with a significative name with this timeout value in case this is used only in this test. Otherwise, this value should be added to the vulnerability detector module.